packages/svelte/package.json

```diff
@@ -19,17 +20,54 @@
   "keywords": [
     "svelte"
   ],
-  "exports": {
-    ".": "./src/index.js",
-    "./server": "./src/server.js"
+  "scripts": {
+    "build": "svelte-kit sync && svelte-package && publint",
+    "prepublishOnly": "npm run build",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch"
   },
-  "main": "src/index.js",
+  "files": [
+    "dist",
+    "!dist/**/*.test.*",
+    "!dist/**/*.spec.*"
+  ],
   "peerDependencies": {
     "svelte": "^3.20.0 || ^4.0.0"
   },
+  "devDependencies": {
+    "@sveltejs/adapter-auto": "^3.2.0",
+    "@sveltejs/kit": "^2.5.5",
+    "@sveltejs/package": "^2.3.0",
+    "@sveltejs/vite-plugin-svelte": "^3.0.2",
+    "publint": "^0.2.7",
+    "svelte": "^4.2.12",
+    "svelte-check": "^3.6.8",
+    "tslib": "^2.6.2",
+    "typescript": "^5.4.3",
+    "vite": "^5.2.7"
+  },
+  "exports": {
```

Comment:
when I run `npm run build` locally, it does not generate the `dist` folder and the corresponding `.js` and `.d.ts` files as described.
Could you please explain how to properly build these files? What settings or additional tools do I need to install to generate the file structure described in the exports configuration?
Are there any other points to note?

Reply:
It should. Do you have any kind of error?

Reply:
Oh sorry! It's my mistake, forgot `run install` before `build`
It works!
Thanks every working.